app.js

```diff
@@ -11,7 +11,82 @@ app.use(express.json());
 app.use(bodyParser.json());
 
 
+app.post('/register', (req, res) => {
+  const userId = randomUUID();
+  const newUser = {
+    id: userId,
+    email: "alex.gmail.com",
+    pass: "[pxnjyfgbcfnm"
+  }
+  res.set({'x-user-id': userId})
+  users.push(newUser);
+  res.status(201).send(users);
```

Comment:
You return ALL users here, it's not ok) Only a new user should be send to the client